src/ui/pages/History/History.tsx

```diff
@@ -120,6 +123,7 @@ function useMinedAndPendingAddressActions({
       actions_search_query: searchQuery,
     },
     {
+      client: useDefiSdkClient(),
```

Comment:
Do we need this if we have global Provider?

Reply:
That's right, not anymore. I'll remove